tests/phpunit/integration/tests/REST_API/Stories_Controller.php

```diff
@@ -235,14 +240,47 @@ public function test_get_items_contributor() {
 		$this->assertArrayHasKey( 'future', $statuses );
 		$this->assertArrayNotHasKey( 'private', $statuses );
 
-		$this->assertEquals( 13, $statuses['all'] );
-		$this->assertEquals( 7, $statuses['publish'] );
-		$this->assertEquals( 3, $statuses['future'] );
-		$this->assertEquals( 3, $statuses['draft'] );
+		$this->assertSame( 7, $statuses['all'] );
+		$this->assertSame( 7, $statuses['publish'] );
+		$this->assertSame( 0, $statuses['future'] );
+		$this->assertSame( 0, $statuses['draft'] );
 
-		$this->assertEquals( 7, $headers['X-WP-Total'] );
+		$this->assertSame( 7, $headers['X-WP-Total'] );
```

Comment:
Looks like `7` is a string here, not an int.

Reply:
Which is why it was an `assertEquals`.

Reply:
Then it should be `$this->assertSame( '7', $headers['X-WP-Total'] );`